tasks/html-css-popup.md

```diff
@@ -25,6 +25,43 @@ The goal of this task is to practice coding HTML & CSS from mockup. You are expe
     makes the content of the popup scrollable
 - mobile view is irrelevant for this task
 
+## Recommendations
+- Make sure that your navigation is built by this structure: 
+``` HTML
```

Comment:
Please add the information why this is recommended.

Recommendation without motivation behind will not be remembered. This can be a link to the article about accessibility or example of problem, that will be solved by this code.

When we just telling "what to do", students will not have mental model to which they can "bind" this information. It will be simply fullfilled and forgotten.

Also, you can met people, who has different opinions and without additional information they can reject recommendation because of their, even falsy, experience.

Reply:
Ok. I agree with you. Updated.